### PR TITLE
Feature/Authentication Swagger #101

### PR DESCRIPTION
**Pull request by wotwot563**
This pull request enables the authentication feature so that we can login with an account 

Closes: #101

### CODE REVIEW

File: API/Configuration/Config.cs [hunk 1]
```diff
@@ -49,13 +49,18 @@ public class Config : IValidatable
         /// </value>
         public IdentityServerConfig IdentityServer { get; set; }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        public SwaggerConfig swaggerConfig { get; set; }
```
Comment (Brend-Smits) [inline API/Configuration/Config.cs:55]: Capital S

File: API/appsettings.Development.json [hunk 2]
```diff
@@ -10,7 +10,11 @@
             "ClientSecret": "Q!P5kqCukQBe77cVk5dqWHqx#8FaC2fDN&bstyxrHtw%5R@3Cz*Z"
         },
         "IdentityServer": {
-            "IdentityUrl": "https://localhost:5005"
+            "IdentityUrl": "http://localhost:5004"
```
Comment (Brend-Smits) [inline API/appsettings.Development.json:13]: Why change this to http instead of https? Locally https works fine as well

**Comment by Brend-Smits** [comment]
Can you also add a new entry to the Changelog please?

File: API/Configuration/Config.cs [hunk 3]
```diff
@@ -49,13 +49,18 @@ public class Config : IValidatable
...
         ///     Validates this instance.
         /// </summary>
         public void Validate()
         {
             Validator.ValidateObject(Frontend, new ValidationContext(Frontend), true);
             Validator.ValidateObject(IdentityServer, new ValidationContext(IdentityServer), true);
+            Validator.ValidateObject(Swagger, new ValidationContext(Swagger), true);
         }
 
         /// <summary>
```
Comment (jcoenen96) [inline API/Configuration/Config.cs:66]: empty summary

File: API/Configuration/Config.cs [hunk 4]
```diff
@@ -110,6 +115,20 @@ public class IdentityServerConfig
 
         }
 
+        /// <summary>
```
Comment (jcoenen96) [inline API/Configuration/Config.cs:118]: Empty summary

File: IdentityServer/Configuration/Config.cs [hunk 5]
```diff
@@ -65,6 +66,7 @@ public void Validate()
             Validator.ValidateObject(Self, new ValidationContext(Self), true);
             Validator.ValidateObject(Api, new ValidationContext(Api), true);
             Validator.ValidateObject(Fhict, new ValidationContext(Fhict), true);
+            Validator.ValidateObject(Swagger, new ValidationContext(Swagger), true);
         }
 
         /// <summary>
```
Comment (jcoenen96) [inline IdentityServer/Configuration/Config.cs:72]: empty summary

File: IdentityServer/Configuration/Config.cs [hunk 6]
```diff
@@ -187,6 +189,13 @@ public class FrontendConfig
 
         }
 
+        public class SwaggerConfig
```
Comment (jcoenen96) [inline IdentityServer/Configuration/Config.cs:192]: No summary

File: CHANGELOG.md [hunk 7]
```diff
@@ -9,7 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- Added this very changelog - [#69](https://github.com/DigitalExcellence/dex-backend/issues/71)
+- Added this very changelog - [#69](https://git.fhict.nl/dex/dex-backend/-/issues/69)
```
Comment (StijnGroenen) [inline CHANGELOG.md:12]: This line in the changelog got changed to the old GitLab URL.
```suggestion
- Added this very changelog - [#71](https://github.com/DigitalExcellence/dex-backend/issues/71)
```

File: API/Configuration/Config.cs [hunk 8]
```diff
@@ -49,13 +49,22 @@ public class Config : IValidatable
         /// </value>
         public IdentityServerConfig IdentityServer { get; set; }
 
+        /// <summary>
+        /// Gets or sets the swagger.
```
Comment (StijnGroenen) [inline API/Configuration/Config.cs:53]: Unclear comment.
Specify what is meant by "the swagger".

**Pull request merged**